### ISSUE TITLE
Update KB article template for cloning in Drupal

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#15328]
## User Story or Problem Statement
The [KB article template for cloning](https://prod.cms.va.gov/kb-article-template-for-cloning) was last updated in 2022 and does not reflect subsequent accessibility, design, and functional best practices. That means any new KB article created from the template may contain inaccessible, non-functional, or non-compliant elements.

The template should be updated to reflect current best practices.

## Description or Additional Context 
[Example of a more recent KB article](https://prod.cms.va.gov/help/cms-basics/outdated-content-audit-tool) that may be a better model

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [ ] Facilities
- [ ] User support
- [ ] Accelerated Publishing

**Comment by xiongjaneg** [comment]
@laflannery Stubbed KB article template ticket for any best practices you may want to add.

**Comment by BerniXiongA6** [comment]
@BerniXiongA6 will bring into Sprint 94 as a Product/Delivery task -- thanks @xiongjaneg

**Comment by laflannery** [comment]
@BerniXiongA6 It might be good to also consider [this defect](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/12693) if CMS will be updating and then possibly using this template to make new articles from (maybe?). Currently this template will result in incorrectly formatted articles due to that defect

**Comment by BerniXiongA6** [comment]
Note to @BerniXiongA6 -- when handling this ticket, reference this ticket: #15357

**Comment by BerniXiongA6** [comment]
Note to self: @BerniXiongA6 Started on this, but we'll need to carry over this ticket into Sprint 96 when there's more capacity. Need to make sure defect is handled when editing KB article template. Will keep all posted. Thanks! cc: @EWashb
